vk_raii_ProgrammingGuide.md

```diff
@@ -33,7 +33,7 @@ Other than the `vk::Device`, you can assign the `vk::raii::Device` to a smart po
 
 Note that, as the vk::raii objects own the actual Vulkan resource, all vk::raii objects are moveable, but not copyable.
 
-For simplicity, in the rest of this document a vk::raii object is always directly instantiated on the stack. Obviously, that�s not essential. You could assign them as well to a std::unique_ptr, a std::shared_ptr, or any other smart pointer or object managing data structure. And you can even assign them to a dumb pointer by using the new operator.
+For simplicity, in the rest of this document a vk::raii object is always directly instantiated on the stack. Obviously, that’s not essential. You could assign them as well to a std::unique_ptr, a std::shared_ptr, or any other smart pointer or object managing data structure. And you can even assign them to a dumb pointer by using the new operator.
```

Comment:
No, I meant it should be `'` apostophe, single quote (https://www.compart.com/de/unicode/U+0027 /https://en.wikipedia.org/wiki/Apostrophe) indicating a missing letter instead of backtick that is used for accents (`¡olé!`)

Reply:
Yep that's what I meant as well. I'm able to see the character now since I changed my font, lol. Will make another commit to fix this.